etl/steps/data/garden/emdat/2023-09-20/natural_disasters.meta.yml

```diff
@@ -1,3 +1,9 @@
+definitions:
```

Comment:
General comment:

The metadata for this dataset needs some revision. The descriptions of some of the indicators might not be self-contained, since they sometimes do not make any reference to Natural disasters.

Take `reconstruction_costs` variable, with description `Reconstruction costs.`. I suppose this should be something like `Reconstruction costs after a Natural disaster` or something along this lines. Otherwise, the indicator on its own might not be understandable.

There are other indicators affected.

Reply:
Improving the metadata of natural disasters is on my to-do list. We were planning to work on this topic page by the end of the year, but it has been moved to the future. I'll leave it like this for now.